app/connectors/IFSBusinessDetailsConnector.scala

```diff
@@ -31,7 +31,7 @@ import scala.concurrent.ExecutionContext
 
 trait IFSBusinessDetailsConnector {
   def getBusinesses(nino: Nino)(implicit hc: HeaderCarrier, ec: ExecutionContext): ApiResultT[api_1171.SuccessResponseSchema]
-  def getBusinessIncomeSourcesSummary(taxYear: TaxYear, nino: Nino, businessId: BusinessId)(implicit
```

Comment:
Is there any reason why we are moving back to primitives String? Cannot we just use .value in the very last momment when we send request?

Reply:
Was to avoid parsing errors from BusinessId in the service but you're right, will change it back